src/torchmetrics/functional/text/rouge.py

```diff
@@ -39,13 +41,40 @@
 ALLOWED_ACCUMULATE_VALUES = ("avg", "best")
 
 
+def _is_internet_connection() -> bool:
```

Comment:
I assume changes come from the commit from another MR.

Reply:
yes, my mistake :)